src/tests/tripTest.js

```diff
@@ -0,0 +1,55 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import dotenv from 'dotenv';
+import app from '../index';
+import mockData from './mock/mockData';
+
+chai.use(chaiHttp);
+chai.should();
+
+dotenv.config();
+const {
+  ...data
+} = mockData.trips;
+
+const {
+  ...newUser
+} = mockData.secondUsers;
+const userToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MywiZW1haWwiOiJqYW1zb24xQGdtYWlsLmNvbSIsInVwZGF0ZWRBdCI6IjIwMTktMTEtMTVUMDg6NDQ6MjUuODI4WiIsImNyZWF0ZWRBdCI6IjIwMTktMTEtMTVUMDg6NDQ6MjUuODI4WiIsImlzVmVyaWZpZWQiOm51bGwsInNpZ251cFR5cGUiOm51bGwsImlhdCI6MTU3MzgwNzQ2NX0.a_OQFNggkrmAObYfr6bUOe4NTPYdYnLRToCVatDMsTk';
+const invalidToken = 'zI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MywiZW1haWwiOiJqYW1zb24xQGdtYWlsLmNvbSIsInVwZGF0ZWRBdCI6IjIwMTktMTEtMTVUMDg6NDQ6MjUuODI4WiIsImNyZWF0ZWRBdCI6IjIwMTktMTEtMTVUMDg6NDQ6MjUuODI4WiIsImlzVmVyaWZpZWQiOm51bGwsInNpZ251cFR5cGUiOm51bGwsImlhdCI6MTU3MzgwNzQ2NX0.a_OQFNggkrmAObYfr6bUOe4NTPYdYnLRToCVatDMsTk';
```

Comment:
you could use a shorter string like jhfgjkhdfhgdhfgjhdfjkhgkljhdfljkhglkjdfhljkghdfjkh for invalid because after all it will still be invalid.